packages/swing-store/test/state.test.js

```diff
@@ -155,82 +155,106 @@ test('persistent kvStore maxKeySize write', async t => {
   await hostStorage.close();
 });
 
-async function testTranscriptStore(t, dbDir) {
-  const exportLog = makeExportLog();
-  const { kernelStorage, hostStorage } = initSwingStore(dbDir, {
-    exportCallback: exportLog.callback,
-    keepTranscripts: true, // XXX need to vary
-  });
-  const { transcriptStore } = kernelStorage;
-  const { commit, close } = hostStorage;
+const testTranscriptStore = test.macro({
```

Comment:
We'd need to load data that includes at least two historical spans, which is possible but not trivial. Given that, it didn't seem worth it because I think we don't actually care that they are left around.